tests/main/install-sideload/task.yaml

```diff
@@ -71,7 +71,15 @@ execute: |
     test-snapd-tools.success
 
     echo "All snap blobs are 0600"
-    test "$( find /var/lib/snapd/{snaps,cache,seed/snaps}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+    test "$( find /var/lib/snapd/{snaps,cache}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+
+    # shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB/systems.sh"
+    # The "seed/" dir is on a FAT partition on uc20 so the permissions are
+    # different here.
+    if ! is_core20_system; then
+        test "$( find /var/lib/snapd/seed/snaps/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
```

Comment:
I think this is not needed due to this test is not executed on ubuntu core

Reply:
The test now is failing in all the ubuntus: https://paste.ubuntu.com/p/tqQDfd7mNM/

Reply:
Thank you!